### PR TITLE
Add Other Details to patient registration

### PR DESCRIPTION
**Pull request by nanfuka**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-registration-app/src/config-schema.ts [hunk 1]
```diff
@@ -1,11 +1,24 @@
 import { Type, validator, validators } from '@openmrs/esm-framework';
 
+let showDetails = false;
```
Comment (hadijahkyampeire) [inline packages/esm-patient-registration-app/src/config-schema.ts:3]: Why this?

File: packages/esm-patient-registration-app/src/config-schema.ts [hunk 2]
```diff
@@ -263,6 +301,107 @@ export const esmPatientRegistrationSchema = {
       ],
       _description: 'Provide ability to configure sex options.',
     },
+
+    maritalStatus: {
+      _type: Type.Array,
+      _elements: {
+        value: {
+          _type: Type.String,
+          _description: 'The value for sex option',
```
Comment (hadijahkyampeire) [inline packages/esm-patient-registration-app/src/config-schema.ts:310]: I don't think this should be `the value for sex option` 🤔, I think it should probably be `the value for marital status.`

**Comment by denniskigen** [comment]
@nanfuka, is this still relevant?

**Comment by nanfuka** [comment]
this pr is nolonger necessary

**Pull request closed without merging**